ci-operator/step-registry/ipi/conf/azure/sharednetwork/ipi-conf-azure-sharednetwork-commands.sh

```diff
@@ -8,17 +8,35 @@ set -o pipefail
 curl -L https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
 
 CONFIG="${SHARED_DIR}/install-config.yaml"
-PATCH=/tmp/install-config-sharednetwork.yaml.patch
 
-azure_region=$(/tmp/yq r "${CONFIG}" 'platform.azure.region')
+if [ -f "${SHARED_DIR}/customer_vnet_subnets.yaml" ]; then
+  RESOURCE_GROUP=$(cat ${SHARED_DIR}/resouregroup)
+  vnet_name=$(cat ${SHARED_DIR}/customer_vnet_subnets.yaml | /tmp/yq r - '.platform.azure.virtualNetwork')
```

Comment:
I do not think this is a correct syntax.
```
[root@preserve-jialiu-ansible ~]# cat test.2
platform:
  azure:
    networkResourceGroupName: os4-common
    virtualNetwork: do-not-delete-shared-vnet-${azure_region}
    controlPlaneSubnet: subnet-1
    computeSubnet: subnet-2
[root@preserve-jialiu-ansible ~]# cat test.2 | yq r - '.platform.azure.virtualNetwork'
[root@preserve-jialiu-ansible ~]# cat test.2 | yq r - 'platform.azure.virtualNetwork'
do-not-delete-shared-vnet-${azure_region}
```

And can we keep the same syntax as how the original code use yq to read region info from install-config.yaml? something like `yq r ${SHARED_DIR}/customer_vnet_subnets.yaml 'platform.azure.virtualNetwork'`